src/forms/inputs.js

```diff
@@ -184,6 +184,12 @@ class ManagedNumberField extends Component {
     var { valid, focused, focusedVal }                = this.state;
     var { value, name, className, format, otherData } = this.props;
 
+    // @todo Bad var name. Find more useful name for this.
+    let id = name;
```

Comment:
Maybe something like `constructedId` or `augmentedId`?

Reply:
Oh, sorry. That wasn't specific and was put in the wrong place. I meant `interval`. Something like `idSuffix` might be useful for that. @todo